src/core/createLogger.js

```diff
@@ -42,19 +42,10 @@ const createLogger = (console, logController, prefix) => {
      * Outputs an error message to the web console.
      * @param {...*} arg Any argument to be logged.
      */
-    error: process.bind(null, "error"),
+    error: process.bind(null, "error")
     /**
      * Creates a new logger with an additional prefix.
      * @param {String} additionalPrefix
      */
```

Comment:
should this comment be removed since the subsequent code has been?

Reply:
Yes, thanks! I'm submit a new PR for that.